src/cephadm/cephadm

```diff
@@ -2338,7 +2339,10 @@ class CephContainer:
             priv = ['--privileged',
                     # let OSD etc read block devs that haven't been chowned
                     '--group-add=disk']
-        if self.ptrace:
+        if self.ptrace and not self.privileged:
+            # if privileged, the SYS_PTRACE cap is already added
+            # in addition, --cap-add and --privileged are mutually
+            # exclusive since podman >= 2.0
             priv.append('--cap-add=SYS_PTRACE')
```

Comment:
if you make this a dedicated PR, we could merge this right away. wdyt?

Reply:
ack

Reply:
https://github.com/ceph/ceph/pull/37133